### ISSUE TITLE
KeyError: 'No serializer found for the User model

### ISSUE THREAD

**Issue by chriamue** [briancappello/flask-unchained#1]
It looks like there are serializers for https://github.com/briancappello/flask-unchained/blob/master/flask_unchained/_code_templates/project/app/models/user.py needed, which is missing.
```bash
pip install git+https://github.com/briancappello/flask-unchained
flask new project testproj # y to all
cd testproj/
pip install -r requirements-dev.txt
flask run
```

> Traceback (most recent call last):
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/bundles/api/hooks/register_model_resources_hook.py", line 30, in attach_serializers_to_resource_cls
    serializer_cls = self.bundle.serializers_by_model[model_name]
KeyError: 'User'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/cli.py", line 56, in cli_create_app
    return AppFactory.create_app(env)
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/app_factory.py", line 75, in create_app
    unchained.init_app(app, env, bundles, _config_overrides=_config_overrides)
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/unchained.py", line 107, in init_app
    run_hooks_hook.run_hook(app, bundles, _config_overrides=_config_overrides)
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/hooks/run_hooks_hook.py", line 34, in run_hook
    hook.run_hook(app, bundles)
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/app_factory_hook.py", line 98, in run_hook
    self.process_objects(app, self.collect_from_bundles(bundles))
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/bundles/api/hooks/register_model_resources_hook.py", line 25, in process_objects
    self.attach_serializers_to_resource_cls(model_name, resource_cls)
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/bundles/api/hooks/register_model_resources_hook.py", line 32, in attach_serializers_to_resource_cls
    raise KeyError(f'No serializer found for the {model_name} model')
KeyError: 'No serializer found for the User model'

**Comment by briancappello** [comment]
Hey, thanks for the report!

I *think* this is a result of poor version-pinning in `setup.py` on my part, can you try this:
```
pip install marshmallow-sqlalchemy==0.14.1
```

(installing v0.6.6 of flask unchained today also required me to pin these versions:)
```
pip install flask-sqlalchemy-unchained==0.3.1 sqlalchemy-unchained==0.4.0 py-meta-utils==0.3.0
```

**Comment by chriamue** [comment]
> pip install marshmallow-sqlalchemy==0.14.1

Works

**Comment by chriamue** [comment]
> pip install flask-sqlalchemy-unchained==0.3.1 sqlalchemy-unchained==0.4.0 py-meta-utils==0.3.0

gives me the warning
```bash
flask-unchained 0.6.6 has requirement py-meta-utils>=0.7.3, but you'll have py-meta-utils 0.3.0 which is incompatible.
```
This error then happens when starting 'flask  run'
```bash
flask run
Traceback (most recent call last):
  File "/home/chriamue/temp/venv/bin/flask", line 7, in <module>
    from flask_unchained.cli import main
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/__init__.py", line 14, in <module>
    from .app_factory import AppFactory
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/app_factory.py", line 8, in <module>
    from .bundle import AppBundle, Bundle
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/bundle.py", line 6, in <module>
    from .flask_unchained import FlaskUnchained
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/flask_unchained.py", line 3, in <module>
    from .unchained import Unchained, unchained
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/unchained.py", line 14, in <module>
    from .di import _ensure_service_name, injectable, _inject_cls_attrs
  File "/home/chriamue/temp/venv/lib/python3.6/site-packages/flask_unchained/di.py", line 4, in <module>
    from py_meta_utils import (AbstractMetaOption, McsArgs, MetaOptionsFactory,
ImportError: cannot import name 'process_factory_meta_options'
```

**Comment by briancappello** [comment]
Did you install Flask Unchained from master? In that case you'll need the latest versions of flask-sqlalchemy-unchained, sqlalchemy-unchained, and py-meta-utils (but still with marshmallow-sqlalchemy pinned to 0.14.1)

**Comment by briancappello** [comment]
This should be fixed in 0.7.0
